### PR TITLE
[WEB] Update search filters

### PR DESCRIPTION
**Pull request by luizeboli**
## Summary

Part of #489

https://www.loom.com/share/21210dada2cb48daa39c0b2ba568d58f

## Relevant technical choices

**As this PR complements PR #583, if you want to fully test it, you should checkout #583 branch and then run `npm run algolia:index` script. So you can checkout back to this branch 😄** 

## QA Steps

1. Read above note
2. Access search page
3. Look at new inputs and try to filter changing them

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/components/Algolia/rangeSliderWithPanel.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+					)}
+				</Handles>
+
+				<Ticks values={ticksValues}>
+					{({ ticks }) => (
+						<div>
+							{ticks.map(({ id, value, percent }, index) => (
+								<div
+									// eslint-disable-next-line react/no-array-index-key
+									key={`${id}_${index}`}
```
Comment (lcnogueira) [inline packages/web/components/Algolia/rangeSliderWithPanel.js:203]: Why not using just the `id` instead id+ index?

File: packages/web/utils/algoliaHelper.js [hunk 2]
```diff
@@ -51,6 +53,22 @@ const createURL = (state) => {
 			.map(encodeURIComponent);
 	}
 
+	if (state?.refinementList?.classification) {
```
Comment (lcnogueira) [inline packages/web/utils/algoliaHelper.js:56]: Much easier to maintain 👌 💯

**Review by lcnogueira — APPROVED** [review]
Looks great 👏

**Review by calaca — APPROVED** [review]
Well done @luizeboli ! :100:

File: packages/web/components/Algolia/rangeSliderWithPanel.js (hunk 1 above)
Comment (luizeboli) [reply packages/web/components/Algolia/rangeSliderWithPanel.js:203]: Hi @lcnogueira 

Adoidado duplicated key if min and max value is the same.

The id here isn't a trully identifier but just the number

**Pull request merged**